UseCase.md

```diff
@@ -1,64 +1,82 @@
-# BTCPay Server Use Case
+# BTCPay Server Use Cases
 
-While primarily build for merchants, BTCPay server is feature-rich software which has plenty of use-cases and solves problems for different users. In this guide, we will show you some of the use-cases of BTCPay. The usage of the software is not limited to these groups.
+Build with the community in mind, BTCPay's ambition is to deliver the best Bitcoin payment service for merchants and their customers, while creating opportunities for everyone to contribute and take parts in its development and growth.
```

Comment:
I don't like this part, it sounds like BTCPay is providing a service of some sort.

Reply:
I see, I rolled back to something very close to what you originally wrote